src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -355,6 +382,60 @@
         }
       }
     },
+    "notification": {
+      "type": "object",
+      "description": "Describes a condition relevant to the tool itself, as opposed to being relevant to a file being analyzed by the tool.",
```

Comment:
Fixed

---
In reply to: [61636449](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61636449) [](ancestors = 61636449)